cmd/advertisement-operator/main.go

```diff
@@ -217,5 +231,6 @@ func main() {
 	}
 	close(c)
```

Comment:
Is c still used somewhere?

Reply:
No, merge it is just a merge oversight. I will drop it.